### PR TITLE
This is a new pull request

### PR DESCRIPTION
**Pull request by jhrozek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # testrepo
 For experiments with git. Nothing to see here.
 
-pull me. push me. rough.
```
Comment (jhrozek) [inline README.md]: please revert

**Comment by jhrozek** [comment]
bla

**Comment by jhrozek** [comment]
blabla

**Comment by jhrozek** [comment]
asdf

**Comment by jhrozek** [comment]
ggg

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
 # testrepo
 For experiments with git. Nothing to see here.
 
-pull me.
```
Comment (jhrozek) [inline README.md]: don't

**Comment by jhrozek** [comment]
zzz

**Comment by jhrozek** [comment]
xxx

**Comment by lslebodn** [comment]
NACK

**Pull request closed without merging**